pycm/pycm_param.py

```diff
@@ -120,6 +120,8 @@
 
 CURVE_NONE_WARNING = "The curve axes contain non-numerical value(s)."
 
+DEPRECATION_WARNING = "`{}` function is deprecated and may be removed in future releases."
```

Comment:
It's okay but I suggest removing that `function` because sometimes it's a method.

```
"`{}` is deprecated and may be removed in future releases."

```

@alirezazolanvari please review this section